### DISCUSSION TITLE
Recommendation for accuracy challenge module

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#23678] [Ideas]
---

**1**. Use a red edge warning to remind players to be vigilant when they receive some "ok" that will cause the acc to be low and out of bounds.

---

**2**. Do not eliminate players in the early stages unless their acc is too low. (For example, with a threshold of 99%, even if a player receives 1 "miss" within 100 notes, they will not be eliminated unless they receive another evaluation other than "great". This also prevents the player from being eliminated when the first note receives "miss".)

It can be roughly calculated as follows:
(x represents the set threshold, y represents the points lost in the game (obtaining an OK increases by 2/3, obtaining an MEH increases by 5/6, and obtaining a miss increases by 1))

If the output of this function is less than x before 1/(1-x) notes, eliminate the player:
1-(y/(1/(1-x))

After 1/(1-x) notes: Keep as is.

---

**3**. Add an option with the function of:  If it is found that the game's acc will never exceed the set minimum acc, the player will be eliminated. (Setting x as the total number of notes can achieve this)

---

To simplify the explanation, all y here are inaccurate because, for example, the slider point.

(machine translation)

**Comment by cdwcgt** [comment]
for 2,3 I think they're all based on the same rationale and https://github.com/ppy/osu/pull/23317
For 1, I'll try to look into
